docs/elements.md

```diff
@@ -704,22 +704,22 @@ generation if two (or more) choices are identical.
-| `none-of-the-above`           | string  | `false` | Add "None of the above" choice. See below for details.                                                                                                                         |
-| `all-of-the-above-feedback`   | string  | —       | Helper text to be displayed to the student next to the `all-of-the-above` option after question is graded if this option has been selected by the student.                     |
-| `none-of-the-above-feedback`  | string  | —       | Helper text to be displayed to the student next to the `none-of-the-above` option after question is graded if this option has been selected by the student.                    |
-| `external-json`               | string  | special | Optional path to a JSON file to load external answer choices from. Answer choices are stored as lists under "correct" and "incorrect" key names.                               |
-| `external-json-correct-key`   | string  | special | Optionally override default json "correct" attribute name when using `external-json` file.                                                                                     |
-| `external-json-incorrect-key` | string  | special | Optionally override default json "incorrect" attribute name when using `external-json` file.                                                                                   |
-| `allow-blank`                 | boolean | false   | Whether or not an empty submission is allowed. If `allow-blank` is set to `true`, a submission that does not select any option will be marked as incorrect instead of invalid. |
+| Attribute                     | Type                                      | Default  | Description                                                                                                                                                                    |
+| ----------------------------- | ----------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `answers-name`                | string                                    | —        | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question.           |
+| `weight`                      | integer                                   | 1        | Weight to use when computing a weighted average score over elements.                                                                                                           |
+| `display`                     | "block", "inline", or "dropdown"          | "inline" | How to display the input field.                                                                                                                                                |
```

Comment:
I think explaining the options here may be a good idea.

Reply:
Explanation added here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/43bdad261b930462316f8fd5e00b162f2b040978